pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>${revision}</version>
 
     <properties>
-        <revision>2.0.1</revision>
+        <revision>2.0.2</revision>
```

Comment:
2.1.0 
we update small version on bug fix , second on new features

Reply:
fixed